lib/v2/sciencenet/user.js

```diff
@@ -0,0 +1,65 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const iconv = require('iconv-lite');
+const timezone = require('@/utils/timezone');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const id = ctx.params.id;
+
+    const rootUrl = 'http://blog.sciencenet.cn';
```

Comment:
```suggestion
    const rootUrl = 'https://blog.sciencenet.cn';
```
Looks like the site also provide HTTPS